community_images/traefik/traefik/.rfignore

```diff
@@ -0,0 +1,5 @@
+sbin/mkmntdirs
+var/
+etc/
+usr/
```

Comment:
Should we check if we need to skip entire directories? Can we try to include coverage so that it is not required?

Reply:
Most of the files are alpine image specific. Keys and certs, with a couple of executables from alpine. As there was very less  reduction in size 3-4% , we agreed to include these, happy to have a look together on these directories.

Reply:
This is unacceptable. We cant whitelist entire root level directories